tasks/deploy.py

```diff
@@ -391,26 +414,20 @@ def deploy_built_artefacts(pr, event_info):
     """
     funcname = sys._getframe().f_code.co_name
 
-    log(f"{funcname}(): deploy for PR {pr.number}")
-
-    deploycfg = config.get_section(DEPLOYCFG)
-
     # verify that the GH account that set label bot:deploy has the
     # permission to trigger the deployment
-    deploy_permission = deploycfg.get(DEPLOY_PERMISSION, '')
-    log(f"{funcname}(): deploy permission '{deploy_permission}'")
+    cfg = read_config(DEPLOYCFG, DEPLOY_PERMISSION)
```

Comment:
`read_config` should be a function that is imported from `tools.config`, which reads a given configuration file at a specified path, and returns the configuration (as a dictionary). The idea is that this replaces the usage of the `read_file` (which is called in the `main` function in `eessi_bot_job_manager.py` and `eessi_bot_job_manager.py`) and `get_section` functions (both can be removed), as follows:

```python
cfg = read_config('app.cfg')
...
if labeler not in cfg[DEPLOYCFG][DEPLOY_PERMISSION]:
...
upload_policy = cfg[DEPLOYCFG][DEPLOY_PERMISSION]
```

That way, we can stop using the global variable `_config` in `tools/config.py`, which is initialized

Reply:
or:

```python
cfg = read_config('app.cfg')
deploy_cfg = cfg[DEPLOYCFG]
...
if labeler not in deploy_cfg[DEPLOY_PERMISSION]: